styles/todo.css

```diff
@@ -0,0 +1,79 @@
+
+main {
+    background-image: url('../assets/todo.jpg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#add-btn {
+    width: 100px;
+    padding: 0;
+    font-size: 60px;
+}
+
+form, #add-btn {
+    position: relative;
+    margin: 30px;
+    border: 5px white solid;
+    border-radius: 10px;
+    height: 100px;
+}
+
+form, #add-btn, #submit-btn {
+    color: white;
+    background-color: rgba(0, 0, 0, 0.788);
+}
+
+form {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 500px;
+    font-size: 20px;
+}
+
+#submit-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 30px;
+    height: 30px;
+    border: 3px white solid;
+    border-radius: 8px;
+    margin-left: 10px;
+    font-size: 25px;
+}
+
+label, input {
+    font-size: 20px;
+}
+
+.dynamic-todo, .dynamic-todo-done {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 500px;
+    height: 50px;
+    border-radius: 8px;
+    margin-top: 15px;
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.dynamic-todo {
+    color: white;
+    background-color: rgba(0, 0, 0, 0.788);
```

Comment:
`opacity: 0.788` is a nifty way to do this if, say, you want to have your colors saved as a variable and adjust just the opacity of different elements